docs/Catalog/Packaging.md

```diff
@@ -0,0 +1,188 @@
+<!--pytest-codeblocks:skipfile-->
+# Packaging Engine
+
+## Overview
+
+The Quilt Packaging Engine in the Quilt Platform allows administrators and
+developers to automate the process of creating Quilt packages from data stored
+in Amazon S3. It serves as a key component in Quilt's SDMS (Scientific Data
+Management System) strategy, enabling automated data ingestion and
+standardization. It consists of:
+
+1. Admin Settings GUI to enable package creation based on notifications from:
+   1. AWS Health Omics
+   2. Nextflow workflows using  `nf-prov`'s WRROC ([Workflow Run
+      RO-Crate](https://www.researchobject.org/workflow-run-crate/)) format
+2. An SQS queue that will process package descriptions
+3. Documentation for creating custom EventBridge rules to invoke that queue
+
+## Admin Settings
+
+The simplest way to enable package creation is through the Admin Settings GUI,
+which supports the following built-in event sources:
+
+![Admin Settings](../imgs/package-admin-gui.png)
+
+### AWS Health Omics
+
+When enabled, this will create a package from the `runOutputUri` provided in a
+`aws.omics` completion event. For example, if the `runOutputUri` is
+`s3://quilt-example/omics-quilt/3395667`, the package will be created in that
+same bucket with the name `omics-quilt/3395667`.
+
+### Workflow Run RO-Crate
+
+When enabled, this will create a package when ingesting any folder containing an
+`ro-crate-manifest.json`.  Ingest happens when the bucket is added to the stack,
```

Comment:
> Ingest happens when the bucket is added to the stack,

no

Reply:
```suggestion
When enabled, this will create a package when indexing any folder containing an
`ro-crate-manifest.json`.  Indexing happens when the bucket is added to the stack,
```

Is indexing a better term? Do we not get these notifications when a bucket is added?

Reply:
when we add a bucket we generate "S3 events" only for indexer queue not for our bus